options.go

```diff
@@ -39,6 +39,10 @@ type Options struct {
 	// Logger is the customized logger for logging info, if it is not set,
 	// default standard logger from log package is used.
 	Logger Logger
+
+	// When DisablePurge is false, a separate goroutine is started and periodically purges expired workers.
```

Comment:
I think the comment here should state what it means when DisablePurge is `true`, not the other way around.